tools/functional-tester/etcd-tester/tester.go

```diff
@@ -122,20 +128,32 @@ func (tt *tester) runLoop() {
 					return
 				}
 			}
-			if !isSameValueInMap(hashes) {
+			if _, ok = isSameValueInMap(hashes); !ok {
```

Comment:
why do we need to do this again? we already know everything is OK. we just need to do the compaction I believe.